doc/rst/language/spec/statements.rst

```diff
@@ -978,17 +978,17 @@ context managers. The syntax of the manage statement is given by
     expression
 
 Classes or records that wish to be used as context managers must
-define two special methods. The code sample below turns a record
-type named ``IntWrapper`` into a context manager and then uses it
-in a manage statement.
+define two special methods, and implement the ``contextManager`` interface.
+The code sample below turns a record type named ``IntWrapper`` into a context
+manager and then uses it in a manage statement.
```

Comment:
I suggest that "implement CM" is fundamental and "define two methods" is a consequence. Also I suggest avoiding the term "special" as I do not know what that means, formally speaking. Also "turn IntWrapper into a context manager" does not make sense because IntWrapper is intended as a CM from the beginning. So:

[the first line unchanged] implement the CM interface by defining two methods enterContext and exitContext. The example below defines a context manager IntWrapper and uses it in a `manage` statement.

Reply:
I've taken a stab at rewording this somewhat.